build/build.sh

```diff
@@ -60,3 +60,13 @@ if [[ $(uname -s) == 'Linux' ]]; then
 fi
 
 popd
+
+# Use a naive substring searching to check if coverage is requested.
+if [[ $@ == *'--config=coverage'* ]]; then
+    # generate coverage for Java
+    pushd source
+    # Set PATH for Java tests
+    PATH=$PATH:`pwd`/bazel-bin/neuropod/multiprocess/
+    bazel coverage --collect_code_coverage --instrumentation_filter='/java[/:]' --combined_report=lcov --coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main //neuropod/bindings/java/...
```

Comment:
You can replace all these flags with a `--config=coverage` to use the flags you added to `.bazelrc`

Reply:
I need to revert .bazelrc. This is what I tried first, I thought that bazel build --config=coverage would work as "bazel coverage" but it didn't. So, my plan - revert .bazelrc and use this command line directly.